dashboard/src/pages/TreeDetails/Tabs/Boots/BootsTab.tsx

```diff
@@ -40,13 +45,21 @@ const BootsTab = ({ treeDetailsLazyLoaded }: BootsTabProps): JSX.Element => {
   const { treeId } = useParams({
     from: '/tree/$treeId',
   });
-  const { tableFilter, diffFilter } = useSearch({
+
+  const searchParams = useSearch({
     from: '/tree/$treeId',
   });
+
+  const { tableFilter, diffFilter } = searchParams;
   const currentPathFilter = diffFilter.bootPath
     ? Object.keys(diffFilter.bootPath)[0]
     : undefined;
 
+  const historyState = useRouterState({ select: s => s.location.state });
+
+  historyState['from'] = RedirectFrom.Tree;
+  historyState['id'] = treeId;
+
```

Comment:
You don't need to get the historyState with `useRouterState` if you are going to overwrite its values anyway. 
```suggestion
const historyState = useMemo(() => ({ id: treeId, from: RedirectFrom.Tree }), [treeId]);
```